### PR TITLE
fix: add fixed date column width in RSS feed

### LINKED ISSUE #657: fixed date column width in RSS feed widget
When using multiple RSS feeds above each other it's visually improper to automatically size the columns of the table (shot below)

image

I suggest to hardwire the width of the left (date) column so that the dates just fit in and leave the rest of the space to the right column. The date is implemented to not break into multiple lines anyways.

### PR DESCRIPTION
**Pull request by TimonRey**
closes #657

### CODE REVIEW

**Review by davifantasia — DISMISSED** [review]
Why is this file included in the changes?

`packages/writing-style/vale-2.1.1`

Also, there is a problem with the actual page, but I'm not sure yet:

Screenshot 2020-09-24 at 13 44 47

**Comment by emmenko** [comment]
The error is

```
The `style` prop expects a mapping from style properties to values, not a string. For example, style={{marginRight: spacing + 'em'}} when using JSX.
```

File: packages/ui-kit/src/components/rss-feeds.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ const RssFeeds = (props) => {
           <tbody>
             {data.items.map((item, index) => (
               <tr key={`${item.title}${index}`}>
-                <td>
+                <td style={`width: ${dimensions.widths.rssFeedDateWidth}`}>
```
Comment (emmenko) [inline packages/ui-kit/src/components/rss-feeds.js:90]:
```suggestion
                <td css={css`width: ${dimensions.widths.rssFeedDateWidth}`}>
```

**Review by emmenko — CHANGES_REQUESTED** [review]
- use either the css prop or a style object (better the css prop)
- remove the binary file
- add a changeset

File: packages/writing-style/.gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 *.zip
 *.tar.gz
+vale-*
```
Comment (emmenko) [inline packages/writing-style/.gitignore:3]: I'm not sure we need this. You just need to remove your binary file from disk, as it's moved to another location.

Comment (TimonRey) [reply packages/writing-style/.gitignore:3]: @nkuehn and I discussed it...

File: .changeset/lovely-suits-judge.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-docs/ui-kit': patch
+'@commercetools-docs/writing-style': patch
```
Comment (emmenko) [inline .changeset/lovely-suits-judge.md:3]: Bumping the writing-style package is not necessary

File: packages/writing-style/.gitignore (hunk 2 above)
Comment (emmenko) [reply packages/writing-style/.gitignore:3]: Hmm ok

**Review by emmenko — APPROVED** [review]
Thanks

**Pull request merged**